docs/grammar/SolidityLexer.g4

```diff
@@ -314,7 +316,7 @@ YulDecimalNumber: '0' | ([1-9] [0-9]*);
 YulStringLiteral:
 	'"' DoubleQuotedStringCharacter* '"'
 	| '\'' SingleQuotedStringCharacter* '\'';
-
+YulHexString: 'hex' (('"' ([0-9a-fA-F][0-9a-fA-F])* '"') | ('\'' ([0-9a-fA-F][0-9a-fA-F])* '\''));
```

Comment:
Is this enough or do we also allow underscores and would need to do this similarly to ``HexString``? Will ``let hex := 42``still be valid? That might be worth a test in any case.

Reply:
This I'd still be wondering about, though.
Is
```
let a := hex"01_02"
```
valid after this PR?
If so, then this will currently still fail the grammar, won't it?

Reply:
I can try to add it back in.